### DISCUSSION TITLE
How do I disable all animations for some specific programs?

### DISCUSSION THREAD

**Discussion by glyh** [hyprwm/Hyprland#2992] [Q&A]
For example I'm using warpd, but when it shows up the animation always waste me about 0.5 sec.

**Comment by YarochkinAnton** [comment]
You can disable animations for a specific app with window rules

`windowrulev2 = noanim, class:^(app)$`

use `hyprctl clients` and find `class` of your application and specify it in the rule

**Reply by glyh** [reply]
Won't work for my case, I run warpd under kitty and got this:
```
Window 55c2d0f454a0 -> warpd --hint ~:
	mapped: 1
	hidden: 0
	at: 5,35
	size: 630,760
	workspace: 2 (2)
	floating: 0
	monitor: 0
	class: kitty
	title: warpd --hint ~
	initialClass: kitty
	initialTitle: fish
	pid: 2703912
	xwayland: 0
	pinned: 0
	fullscreen: 0
	fullscreenmode: 0
	fakefullscreen: 0
	grouped: 0
	swallowing: 0

```
